pyrisco/local/panels.py

```diff
@@ -69,6 +81,7 @@ def _rw512_capabilities(firmware):
   'RW232': _rw232_capabilities,
   'RW332': _rw332_capabilities,
   'RP432': _rw432_capabilities,
+  'RP432MP': _rw432mp_capabilities,
```

Comment:
shouldn't it be `_rp432mp_capabilities`?

Reply:
Yes, it looks like I was getting confused with the `rp` and `rw` naming schemes 😅